pages/technologies/java-junit.mdx

```diff
@@ -122,36 +124,65 @@ The moment you select the Java (JUint), the following code should appear in your
 
 ```java
 import static org.junit.jupiter.api.Assertions.*;
-import java.lang.reflect.*;
 
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.BeforeAll;
-import org.junit.jupiter.api.BeforeEach;
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.io.PrintStream;
 
-import org.junit.jupiter.api.MethodOrderer;
-import org.junit.jupiter.api.TestMethodOrder;
-import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.*;
 import org.junit.jupiter.api.MethodOrderer.OrderAnnotation;
+import java.io.ByteArrayOutputStream;
+import java.lang.reflect.Method;
 
-import java.util.Arrays;
 
-@TestMethodOrder(OrderAnnotation.class)
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
 class TestFile {
+
+    // IO Streams
+    private static final  ByteArrayOutputStream out = new ByteArrayOutputStream();
```

Comment:
Suggestion: we should keep the I/O testing block and the unit/integration testing with reflection API in separate blocks (as discussed, we might just implement a GUI based I/O testing soon so that block would become redundant)

Reply:
Alright!
Removed docs for stdin and stdout.
Commit: 0f7ee9716897ece16fe8ac5ee4c0bbdc2045b37c